pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if (echo "$OSTYPE" | grep -qs 'darwin'); then
```

Comment:
@luiscoms isn't it sufficient to have the change you made in line 132? (without having to call docker-machine ip). Afaik containers can communicate with each other using the IP that Docker gives them, even if you can't access them from the host Mac.